judging-algorithm/README.md

```diff
@@ -0,0 +1,43 @@
+Judging Algorithm Usage:
+
+1) NAVIGATE TO THIS FOLDER
+
+In your terminal, `cd` in to `/judging-algorithm` within this `mono-repo` repository.
+
+Run `npm install` so that you can run the script.
+
+2) UPLOAD YOUR CSV FILES
+
+Upload the following 3 CSV files in to the `/judging-algorithm` folder:
+a. judges.csv
+b. hackers.csv
```

Comment:
add steps on how to get the hackers csv from devpost  (reference the read me from last year's judging algorithm) 
<img width="1192" alt="Screenshot 2023-02-08 at 8 54 23 PM" src="https://user-images.githubusercontent.com/59738880/217696795-95d81967-92be-440f-a115-15d417b0f80b.png">